api/location.ts

```diff
@@ -309,15 +309,14 @@ export class Location {
 
   setLightGroup(groupId: string, on: boolean, durationSeconds = 60) {
     this.restClient.request<any>({
-      method: 'POST',
+      method: 'post',
```

Comment:
Looks like it works just fine if you leave all of these as `POST`/`GET` etc, which should help cut down on the size of your PR

Reply:
Actually it was the TypeScript compiler that complained about the uppercase method names...